### PR TITLE
docs: mention CONTRIBUTING.md in README

### PR DESCRIPTION
**Pull request by ykdojo**
## Test plan

Manually check README.md

### CODE REVIEW

**Review by codycwiseman — APPROVED** [review]

**Comment by codycwiseman** [comment]
As seen on stream.

Good for both humans and AI

File: vscode/README.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ Cody Enterprise can search context from your entire remote codebase using Source
 - [Discord](https://discord.gg/s2qDtYGnAE)
 - [Twitter (@sourcegraph)](https://twitter.com/sourcegraph)
 
+## Set Up & Contribute
+
+Please take a look at CONTRIBUTING.md for more information about how to set up this project and contribute.
```
Comment (dominiccooney) [inline vscode/README.md:78]:
```suggestion
Please take a look at [CONTRIBUTING.md](CONTRIBUTING.md) for more information about how to set up this project and contribute.
```

**Review by dominiccooney — APPROVED** [review]
Looks good, please make it a link.

**Comment by ichim-david** [comment]
@ykdojo, highjacking this pull request a bit because I see that you guys want to bring more awareness to the Contributing docs.
I have a few comments regarding the Contributions docs and their accuracy as I ran into some issues as I tried to use them as guidelines for my first pull request https://github.com/sourcegraph/cody/pull/5874.
Hopefully, they make sense and are useful as community feedback on pain points dealing with the contribution guidelines.

Stuff such as:
1. The docs mention that to test the web dev the sourcegraph server is set to sourcegraph.com
https://github.com/sourcegraph/cody/blob/main/doc/dev/index.md#use-web-build-for-quick-ui-iteration
It used to be that value if I  visit
https://sourcegraph.com/github.com/sourcegraph/cody@c9e483df12dc7547dcdb19abece034c42e0f9039/-/blob/web/demo/App.tsx?L17-19
but nowadays it is set to sourcegraph.sourcegraph.com which is a bummer since I don't have access to it as an open-source contributor.
https://sourcegraph.com/github.com/sourcegraph/cody/-/blob/web/demo/App.tsx?L14
2. asdf install doesn't work, running it gives the warning:
Install plugins first to be able to install tools
Here the docs should clarify what tools need to be installed and that you might have them if you look at this file
https://github.com/sourcegraph/cody/blob/main/.tool-versions
"Run asdf install (if needed, run asdf plugin add NAME for any missing plugins)" It's not clear what this means and how you find out what plugins you need to add and this is why I mention linking to the .tool-versions file.
3. There isn't any mention of pnpm biome command which you should run as part of testing since this process runs on CI and will fail if you make formating changes such as on vscode/package.json.
I think it should be mentioned here https://github.com/sourcegraph/cody/blob/main/vscode/CONTRIBUTING.md

**Comment by ykdojo** [comment]
@ichim-david thank you for mentioning these - I'll try & circle back to this.

For making it a link, I realized there are multiple contribution docs now, and there is also a contribution section that I missed previously.

Given these, I'll close this PR for now without merging it, and I'll revisit it when it makes sense.

**Pull request closed without merging**